mono/metadata/marshal.c

```diff
@@ -6192,19 +6199,24 @@ mono_icall_end (MonoThreadInfo *info, HandleStackMark *stackmark, MonoError *err
 		mono_error_set_pending_exception (error);
 }
 
+extern char const * const mono_handle_track_owner = NULL;
+
 MonoObjectHandle
 mono_icall_handle_new (gpointer rawobj)
 {
+#ifdef MONO_HANDLE_TRACK_OWNER
+	char const * const mono_handle_track_owner = "<marshal args>";
+#endif
 	return MONO_HANDLE_NEW (MonoObject, (MonoObject*)rawobj);
 }
 
 gpointer
 mono_icall_handle_new_interior (gpointer rawobj)
```

Comment:
Need to see why this was done.

Reply:
I think the goal here was to let mono_icall_handle_new override the owner string.